commit c64b01721bddc3e41bb3b2deed72f0d520f3c2de (refs/changes/55/1455/7)
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2017-02-08T20:44:44+00:00 (2 years, 8 months ago)
    
    HEAD-2347 bump default COAL DRAM to 8G
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/bin/build-coal-image b/bin/build-coal-image
index a53971d2..8ed5cbda 100755
--- a/bin/build-coal-image
+++ b/bin/build-coal-image
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright 2015 Joyent, Inc.
+# Copyright 2017 Joyent, Inc.
 #
 
 ROOT=$(cd $(dirname $0)/../; pwd)
@@ -278,10 +278,10 @@ function create_output
 
     coal_memsize=$(build_spec coal-memsize)
     if [[ -n ${coal_memsize} ]]; then
-        if [[ ${coal_memsize} < 4096 ]]; then
+        if [[ ${coal_memsize} < 6144 ]]; then
             # BEGIN BASHSTYLED
             echo "* * *"
-            echo "* Warning: Your COAL memory size (coal-memsize) is set to <4096 MiB."
+            echo "* Warning: Your COAL memory size (coal-memsize) is set to <6144 MiB."
             echo "* That is likely to cause problems in your COAL. Consider a larger"
             echo "* value for 'coal-memsize' in 'build.spec.local'."
             echo "* * *"
diff --git a/build.spec b/build.spec
index 760cc5aa..874966cc 100644
--- a/build.spec
+++ b/build.spec
@@ -1,7 +1,7 @@
 {
   "vmware_version": 5,
   "build-tgz": "true",
-  "coal-memsize": 4096,
+  "coal-memsize": 8192,
   "coal-enable-serial": true,
   "no-rabbit": true,
   "clean-cache": true,
